kyber_gt.go

```diff
@@ -29,8 +30,9 @@ func (k *KyberGT) Equal(kk kyber.Point) bool {
 const gtLength = 576
 
 func (k *KyberGT) Null() kyber.Point {
-	var zero [gtLength]byte
-	k.f, _ = bls12381.NewGT().FromBytes(zero[:])
+	// One since we deal with Gt elements as a multiplicative group only
+	// i.e. Add in kyber -> mul in kilic/, Neg in kyber -> inverse in kilic/ etc
+	k.f = bls12381.NewGT().New()
```

Comment:
Does that actually work?
See: 
https://github.com/drand/kyber-bls12381/blob/492f41b80879461b34f62d85da40dbefc529c6ea/kyber_group.go#L129-L131

Reply:
Ah never mind, it's from `kilic`!